app/konnect/runtime-manager/kic.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Kong Ingress Controller for K8 Association
```

Comment:
```suggestion
title: Kong Ingress Controller for Kubernetes Association
```

Reply:
based on discussion, we're replacing "K8" as internal shorthand used when referring to the full product name.